easybuild/tools/filetools.py

```diff
@@ -122,28 +122,28 @@
 
 EXTRACT_CMDS = {
     # gzipped or gzipped tarball
-    '.gtgz':    "tar xzf %(filepath)s",
+    '.gtgz':    "tar x%(timestamp_tar)szf %(filepath)s",
     '.gz':      "gunzip -c %(filepath)s > %(target)s",
-    '.tar.gz':  "tar xzf %(filepath)s",
-    '.tgz':     "tar xzf %(filepath)s",
+    '.tar.gz':  "tar x%(timestamp_tar)szf %(filepath)s",
+    '.tgz':     "tar x%(timestamp_tar)szf %(filepath)s",
     # bzipped or bzipped tarball
     '.bz2':     "bunzip2 -c %(filepath)s > %(target)s",
-    '.tar.bz2': "tar xjf %(filepath)s",
-    '.tb2':     "tar xjf %(filepath)s",
-    '.tbz':     "tar xjf %(filepath)s",
-    '.tbz2':    "tar xjf %(filepath)s",
+    '.tar.bz2': "tar x%(timestamp_tar)sjf %(filepath)s",
+    '.tb2':     "tar x%(timestamp_tar)sjf %(filepath)s",
+    '.tbz':     "tar x%(timestamp_tar)sjf %(filepath)s",
+    '.tbz2':    "tar x%(timestamp_tar)sjf %(filepath)s",
     # xzipped or xzipped tarball
-    '.tar.xz':  "unxz %(filepath)s --stdout | tar x",
-    '.txz':     "unxz %(filepath)s --stdout | tar x",
+    '.tar.xz':  "unxz %(filepath)s --stdout | tar x%(timestamp_tar)s",
+    '.txz':     "unxz %(filepath)s --stdout | tar x%(timestamp_tar)s",
     '.xz':      "unxz %(filepath)s",
     # tarball
-    '.tar':     "tar xf %(filepath)s",
+    '.tar':     "tar x%(timestamp_tar)sf %(filepath)s",
     # zip file
-    '.zip':     "unzip -qq %(filepath)s",
+    '.zip':     "unzip -qq %(timestamp_zip)s %(filepath)s",
```

Comment:
Avoid double space by using `unzip -qq %(timestamp_zip)s%(filepath)s` and including the space in `ts_zip`